packages/runtime/src/public/createApp.ts

```diff
@@ -0,0 +1,93 @@
+// Note: We should be careful of what we import in this module, we don't want to load the entire runtime here
+// until createApp is called, since we set some globals here, which will be read by other runtime modules
+import '@mobilestack-xyz/runtime/src/missingGlobals'
+import Config from 'react-native-config'
+import DeviceInfo from 'react-native-device-info'
+import { ToggleableOnboardingFeatures } from 'src/onboarding/types'
+
+// This will evolve. We should be mindful of breaking changes.
+// This structure should scale well as we add more features
+// and makes it clear what's core configuration vs optional features.
+export interface PublicAppConfig {
+  registryName: string
+  displayName: string
+  deepLinkUrlScheme: string
+  iosAppStoreId?: string
+
+  // Optional features/capabilities
+  features?: {
+    firebase?: boolean
+    sentry?: {
+      clientUrl: string
+    }
+    // TODO: what's the marketing name for this?
+    cloudBackup?: {
+      auth0Domain: string
+      auth0ClientId: string
+      web3AuthClientId: string
+    }
+    onboarding?: {
+      cloudBackup?: boolean
+      cloudBackupSetupInOnboarding?: boolean
+      enableBiometry?: boolean
+      protectWallet?: boolean
+    }
```

Comment:
Here, onboarding will still be enabled if not set, but this config will allow to setup what steps are used.
We'll document each feature properly to remove confusion about the default behavior when not set.